src/PlatformServices.Interface/PlatformServices.Interface.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface</RootNamespace>
+    <AssemblyName>Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface</AssemblyName>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <IsShipping>true</IsShipping>
```

Comment:
What's the goal of this property?

Reply:
Arcade syntax. It marks a package as a package we ship to customers and tags it correctly in DARC. It also generates and verifies SBOM files.